annotationframeworkclient/infoservice.py

```diff
@@ -28,12 +28,12 @@ def format_neuroglancer_graphene(objurl):
 def format_cloudvolume(objurl):
     qry = urlparse(objurl)
     if qry.scheme == 'gs':
-        objurl_out = 'https://storage.googleapis.com/{}{}'.format(qry.netloc,
+        objurl_out = 'graphene://https://storage.googleapis.com/{}{}'.format(qry.netloc,
```

Comment:
I'm not sure this is right... Graphene formats need to be backed by a pcg address

Reply:
Agreed. I took a step back and made three different output formatting schemes, one for literal fields (no formatting), one for precomputed fields (precomputed://gs:// vs precomputed://https://) and one for graphene fields (graphene://https:// for all). Now the user shouldn't need to remember which is which, just the target of the call (neuroglancer, cloudvolume, or the literal value).